diff --git a/dttv/jni/Android.mk b/dttv/jni/Android.mk
index e92027c..39d23ec 100644
--- a/dttv/jni/Android.mk
+++ b/dttv/jni/Android.mk
@@ -19,7 +19,7 @@ LOCAL_LDLIBS    := -llog -lz
 #control
 ENABLE_OPENSL = yes
 ENABLE_AUDIOTRACK = no
-ENABLE_ANDROID_OMX = yes
+ENABLE_ANDROID_OMX = no
 ENABLE_OPENGL_V2 = yes
 ENABLE_OPENGL_V1 = no
 ENABLE_ANDROID_AE = no
diff --git a/dttv/jni/Application.mk b/dttv/jni/Application.mk
index ecd286f..d81caf7 100644
--- a/dttv/jni/Application.mk
+++ b/dttv/jni/Application.mk
@@ -1 +1 @@
-APP_STL := gnustl_shared
+#APP_STL := gnustl_shared
diff --git a/dttv/src/dttv/app/DtPlayer.java b/dttv/src/dttv/app/DtPlayer.java
index e513506..e48c251 100644
--- a/dttv/src/dttv/app/DtPlayer.java
+++ b/dttv/src/dttv/app/DtPlayer.java
@@ -99,7 +99,7 @@ public class DtPlayer {
 	}
 	
 	static{
-		System.loadLibrary("gnustl_shared");
+		//System.loadLibrary("gnustl_shared");
 		System.loadLibrary("dtp");
 		System.loadLibrary("dtp_jni");		
 	}
